location/models.py

```diff
@@ -33,6 +33,27 @@ def url_encoded_full_address(self):
     def google_map_url(self):
         return f"https://www.google.com/maps/search/?api=1&query={self.url_encoded_full_address}"
 
+    @property
+    def representative_image(self):
+        return self.apartment_set.exclude(image=None).first().image
+
+    @property
+    def representative_image_or_placeholder(self):
```

Comment:
Will it conflict with this function in #336 ?
![image](https://user-images.githubusercontent.com/18141004/70382895-ff90e800-1931-11ea-931d-fe659ab689c8.png)

Reply:
Let's merge #336 first and let me handle conflict

Reply:
Thank you! I'll let you know as soon as it is merged.

Reply:
@KShih Updated!